### DISCUSSION TITLE
进度条卡顿，2.2.1版本

### DISCUSSION THREAD

**Discussion by 999thtest** [Molunerfinn/PicGo#672] [Q&A]
github的，成功上传了一张，但不知道为什么成功的，接下来就一直卡顿。搞不懂啊，拜托啦救救我把谢谢
[image](https://user-images.githubusercontent.com/83920727/117583977-d71ba780-b13c-11eb-9918-5cae6bd4b2c6.png)

**Comment by Molunerfinn** [comment]
请升级版本
